src/resources/resources.css

```diff
@@ -83,7 +83,46 @@
 
 .pagination-container {
 	margin: auto;
+}
+
+/* List Group Buttons To List Items */
```

Comment:
The content seems to be off with the tab bar in width. I will fix the width of tab bar and margin for resources container

Reply:
what do you mean by off? Let me what know what it should be and I can fix it too

Reply:
I mean the horizontal padding of `resources-content` is too spacious, so it should reduced. Thanks by the way :)

Reply:
Okay I'll adjust that!